src/registrar/templates/401.html

```diff
@@ -5,8 +5,8 @@
 {% block title %}{% translate "Unauthorized | " %}{% endblock %}
 
 {% block content %}
-<main id="main-content" class="grid-container {% if is_widescreen_mode %} grid-container--widescreen{% endif %}">
-  <div class="grid-row grow-gap"> 
+<main id="main-content" class="grid-container {% if is_widescreen_mode %} grid-container--widescreen {% endif %}">
```

Comment:
That space that was added back at the end will translate to the markup. We do not want extra spaces in the class value string.